### PR TITLE
[INLONG-5603][DataProxy] The in charges person and other info are not reported

### LINKED ISSUE #5603: [Improve][DataProxy] The in charges person and other info are not reported
### Description

The in charges person and other info are not reported when the node is automatically registered.

### InLong Component

InLong DataProxy

### Are you willing to submit PR?

- [x] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gosonzhang**
- Fixes #5603

### CODE REVIEW

**Review by healchow — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by GanfengTan — APPROVED** [review]

File: inlong-dataproxy/dataproxy-docker/dataproxy-docker.sh [hunk 1]
```diff
@@ -25,6 +25,8 @@ sed -i "s/manager.hosts=.*$/manager.hosts=${MANAGER_OPENAPI_IP}:${MANAGER_OPENAP
 sed -i "s/audit.proxys=.*$/audit.proxys=${AUDIT_PROXY_URL}/g" "${common_conf_file}"
 sed -i "s/proxy.report.ip=.*$/proxy.report.ip=${local_ip}/g" "${common_conf_file}"
 sed -i "s/proxy.cluster.tag=.*$/proxy.cluster.tag=${CLUSTER_TAG}/g" "${common_conf_file}"
+sed -i "s/proxy.cluster.name=.*$/proxy.cluster.name=${$CLUSTER_NAME}/g" "${common_conf_file}"
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-docker/dataproxy-docker.sh:28]: Please add those configs in the `Dockerfile`.

Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-docker/dataproxy-docker.sh:28]: done

**Review by healchow — APPROVED** [review]

File: inlong-dataproxy/dataproxy-docker/Dockerfile [hunk 2]
```diff
@@ -29,6 +29,9 @@ ENV MANAGER_OPENAPI_PORT=8083
 ENV AUDIT_PROXY_URL=127.0.0.1:10081
 # pulsar or tubemq
 ENV MQ_TYPE=pulsar
+ENV CLUSTER_TAG=''
+ENV CLUSTER_NAME=default
+ENV CLUSTER_IN_CHARGES=admin
```
Comment (dockerzhang) [inline inlong-dataproxy/dataproxy-docker/Dockerfile:34]: keep the default same with common.properties
```
proxy.cluster.name=default_dataproxy
proxy.cluster.tag=default_cluster
proxy.cluster.inCharges=admin
```

Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-docker/Dockerfile:34]: done

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**